ADVANCED-TOPICS.md

```diff
@@ -208,3 +208,9 @@ Let's assume you want to create a new preset for `ES8`.
     'plugin:es5/no-es2016'
   ],
   ```
+
+### Adding polyfills info for the [compat plugin](https://github.com/amilajack/eslint-plugin-compat)
+
+This will vary depending on your app, mostly of your usage of [ember-cli-babel-polyfills](https://github.com/pzuraq/ember-cli-babel-polyfills).
+
+Please refer to https://github.com/amilajack/eslint-plugin-compat#adding-polyfills.
```

Comment:
Worth documenting excluding tests from IE compat linting

Reply:
We should use the same polyfills everywhere and see that as an exception :)

Reply:
@saintsebastian if we're aiming at being able to run our tests on ie11, then we should make sure their syntax is compatible. @xcambar thoughts?

Reply:
Testing under IE11 is a silver bullet, I don't think we'll ever do it (before IE11 reaches EOL, that is).
Nonetheless, being syntactically consistent is a huge win and it probably is not that costly to maintain/update... no?

In a word, I'd prefer to keep the rules for tests as well.